src/core_modules/capture-core/reducers/descriptions/workingLists/workingLists.reducerDescription.js

```diff
@@ -457,20 +456,31 @@ export const workingListsColumnsOrderDesc = createReducerDescription({
 
 export const workingListsContextDesc = createReducerDescription({
     /*
-    Setting context on init (not on success anymore) because it makes sense for the loading effect.
-    The meaning is slightly changed though, having a context now implies that a request for events was done for this context,
-    not that events was successfully retrieved for this context.
+    Setting context on fetch/init (not on success anymore) because it makes sense for the loading effect.
```

Comment:
```suggestion
    Setting context on templates fetch and list view init (not on success anymore) because it makes sense for the loading effect.
```

Reply:
Since it applies for both reducers but not for the third we can be a little bit more explicit here.